.gitignore

```diff
@@ -1,2 +1,3 @@
 
 PickMeUp/PickMeUp/keys.plist
+PickMeUp/.DS_Store
```

Comment:
Love it, keep adding other files here. You'll probably want to add xcuserdata as well